test/unit/storage_async_test.rb

```diff
@@ -286,23 +290,28 @@ def test_acl_tls
     }
     storage = StorageAsync::Client.send :new, Storage::Helpers.config_with(config_obj)
     assert_client_config(config_obj, storage)
+  ensure
+    [ca_file, cert, key].each(&:unlink)
   end
 
   private
 
   def create_certs(alg)
     ca_cert_file = Tempfile.new('ca-root-cert.pem')
     ca_cert_file.write(create_cert.to_pem)
-    ca_cert_file.rewind
+    ca_cert_file.flush
+    ca_cert_file.close
```

Comment:
I'm not sure if closing will trigger auto-deleting the file. Just not sure. Probably `flush` is enough. But if it works, that's fine with me.

Reply:
I also thought that, but I would say the file is not destroyed until the variable is garbage-collected, and since we are returning the variables to the calling method, they should stay. The tests pass.